spikes/xml/claim.xml

```diff
@@ -0,0 +1,237 @@
+        <!--Date and time (UTC) claim last submitted-->
+        <submitted_at>2016-09-29T13:11:43Z</submitted_at>
+        <!--Date and time claim originally submitted-->
+        <originally_submitted_at>2016-09-29T13:11:43Z</originally_submitted_at>
+        <state>Authorised</state>
+        <!--Time claim authorised in UTC-->
+        <authorised_at>2016-09-29T13:11:43Z</authorised_at>
+    </claim_details>
+
+
+    <case_details>
+        <!--Case type is a code which identifies the case type to the CCR/CCLF systems -->
+        <case_type>XX1111</case_type>
+        <!--- We can send the name of the court or a code which identifies the court to CCR/CCLF -->
+        <court>Aylesbury Crown</court>
+        <case_number>A12345678</case_number>
+        <!--- We can send the name of the court or a code which identifies the court to CCR/CCLF -->
+        <transfer_court>Reading Crown</transfer_court>
+        <transfer_case_number>Z12345678</transfer_case_number>
+        <date_case_started>2016-07-01</date_case_started>
+        <date_trial_concluded>2016-07-06</date_trial_concluded>
+        <estimated_trial_length>4</estimated_trial_length>
+        <actual_trial_length>5</actual_trial_length>
+        <date_trial_fixed_notice>2016-07-06</date_trial_fixed_notice>
+        <date_trial_fixed>2016-07-06</date_trial_fixed>
+        <date_trial_cracked>2016-07-06</date_trial_cracked>
+        <date_trial_cracked_at_third>2016-07-06</date_trial_cracked_at_third>
+        <date_retrial_started>2016-07-06</date_retrial_started>
+        <date_retrial_concluded>2016-07-06</date_retrial_concluded>
+        <retrial_estimated_length>4</retrial_estimated_length>
+        <retrial_actual_length>6</retrial_actual_length>
+        <effictive_pcmh_date>2016-07-06</effictive_pcmh_date>
+        <legal_aid_transfer_date>2016-07-06</legal_aid_transfer_date>
+        <!--- sourece can be Web, Api, Json -->
+        <source>Web</source>
+        <fees_total>3577.33</fees_total>
+        <expenses_total>840.22</expenses_total>
+        <claim_total>4417.55</claim_total>
+        <disbursements_total>6772.33</disbursements_total>
+        <vat_amount>350.66</vat_amount>
+        <cms_number>X67676767</cms_number>
+        <providers_reference>PSR-001/2016/5/88</providers_reference>
+        <evidence_checklist>
+            <!--Doc types can by one of the following (zero or more may be specified:-->
+            <!--'Representation order'-->
+            <!--'LAC1 - memo of conviction'-->
+            <!--'Committal bundle front sheets'-->
+            <!--'A copy of the indictment'-->
+            <!--'Order in respect of judicial apportionment'-->
+            <!--'Expenses invoices'-->
+            <!--'Hardship supporting evidence'-->
+            <!--'Details of previous fee advancements'-->
+            <!--'Justification for out of time claim'-->
+            <doc_type>Representation order</doc_type>
+            <doc_type>Expenses invoices</doc_type>
```

Comment:
the above section is just a checklist of attached evidence types.  We probably don't need to send that.

The actual evidence documents  are in  the documents section, which does contain URI